### PR TITLE
Handle limit reached error

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it handles `LimitReachedError` in graphql protocol.

**PR intended to be tested with API branch:** 

### Screenshots
Screenshot 2021-02-24 at 12 27 34

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [x] Changes are mentioned in the changelog.
1. [ ] The changes are tested in different browsers (Chrome, Firefox, Safari).
1. [ ] The changes are tested in light and dark mode.

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-2180 Handle limitation errors in saleor-dashboard](https://app.clickup.com/t/2549495/SALEOR-2180)

File: src/intl.ts [hunk 1]
```diff
@@ -55,6 +55,9 @@ export const commonMessages = defineMessages({
   lastName: {
     defaultMessage: "Last Name"
   },
+  limitReached: {
+    defaultMessage: "Reached limit for this plan"
```
Comment (patrys) [inline src/intl.ts:59]: It should explicitly say: "Already at the limit. Unable to create more users." No need to refer to "this plan."

**Review by patrys — CHANGES_REQUESTED** [review]
In the screenshot, the error is a toast hidden behind the overlay. Ideally, it should be presented within the modal.

**Comment by dominik-zeglen** [comment]
@patrys should we completely remove toast?

Screenshot 2021-02-25 at 10 14 46

**Comment by jwm0** [comment]
> @patrys should we completely remove toast?
> 
> Screenshot 2021-02-25 at 10 14 46

I'd keep the toast to be consistent across other gql errors but move its z-index so it's not hidden by the modal backdrop

**Review by jwm0 — APPROVED** [review]

**Pull request merged**